public/app/plugins/datasource/mysql/sqlCompletionProvider.ts

```diff
@@ -0,0 +1,190 @@
+import {
```

Comment:
Autocomplete doesn't work for columns. It suggests the database again.

https://user-images.githubusercontent.com/13729989/178792496-90a159d9-11bf-48d0-b863-00bd233f5c4d.mp4